enable/tools/drag_tool.py

```diff
@@ -62,7 +62,7 @@ class DragTool(BaseTool):
 
     # private property to hold the current list of KeySpec instances of the
     # cancel keys
-    _cancel_keys = Property(List(KeySpec), depends_on="cancel_keys")
+    _cancel_keys = Property(List(KeySpec), observe="cancel_keys")
```

Comment:
Do you need to do anything special if you're observing a `List` trait?

Reply:
Only if you are observing contents of the list, e.g. `container.value` used to listen for changes to the value trait of any item in the container, or listening to `container[]`, or `container_items`.  If you are just observing the list itself, you can leave as is

Example from the pyface changes 
<img width="1583" alt="Screen Shot 2021-03-02 at 11 50 13 AM" src="https://user-images.githubusercontent.com/36972686/109691959-95272180-7b4d-11eb-9f30-eae185c39f17.png">

Reply:
Cool. Thanks for clarifying